frontend-web/webclient/app/Files/Synchronization.tsx

```diff
@@ -0,0 +1,149 @@
+            }
+
+        } else {
+            if (syncFolder) {
+                console.log(syncFolder.id);
+                await invokeCommand(SyncFolderApi.remove(bulkRequestOf({ id: syncFolder.id })));
+                setUcloudDeviceId(undefined);
+                setSyncFolder(undefined);
+            }
+        }
+    };
+
+    return <>
+        <SelectableTextWrapper>
+            <Tab selected={!manageDevices} onClick={() => setManageDevices(false)}>Synchronize folder</Tab>
+            <Tab selected={manageDevices} onClick={() => setManageDevices(true)}>Manage devices</Tab>
+        </SelectableTextWrapper>
+        {manageDevices ? (
+            <>
+                <ResourceBrowse
+                    header={<></>}
+                    api={SyncDeviceApi}
+                    onSelect={onDeviceSelect}
+                    embedded={true}
+                    onInlineCreation={((text, product, cb) => ({
+                            product: {id: product.name, category: product.category.name, provider: product.category.provider},
+                            deviceId: text
+                        })
+                    )}
+                    showProduct={false}
+                />
+            </>
+        ) : (
+            <>
+                <Box mt="30px">To use the synchronization feature you have to set up a local instance of <a href="https://syncthing.net">Syncthing</a> on your device and add your device ID in Manage devices.</Box>
+                <Box mt="30px" mb="30px">
+                    <Label>
+                        <Flex>
+                            <Toggle
+                                scale={1.5}
+                                activeColor="--green"
+                                checked={syncFolder !== undefined}
+                                onChange={() => toggleSyncFolder()}
+                            />
+                            <TextSpan ml={20} fontSize={2}>Add {file.id} to synchronization</TextSpan>
+                        </Flex>
+                    </Label>
+
+                </Box>
+
+                {ucloudDeviceId ? (
+                    <>
+                        <Box mb="20px">
+                            The folder is being synchronized with your devices from Device ID:
+                        </Box>
+                        <Flex>
+                        <Input readOnly={true} value={ucloudDeviceId} />
+                        <Button onClick={() => copyToClipboard({
+                            value: ucloudDeviceId,
+                            message: "Copied " + ucloudDeviceId + " to clipboard"
+                        })}>Copy</Button>
+                        </Flex>
+                        <Box mt="20px">
+                            Add this as a remote device to your local instance of Syncthing to begin synchronization.
+                        </Box>
+                    </>
+                ) : (<></>)}
```

Comment:
I think this will work just as well.
```suggestion
                ) : null}
```